src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -298,14 +298,17 @@ public void disconnected(SchedulerDriver driver) {
         LOGGER.info("Framework disconnected!");
     }
 
+    private void declineOffer(Offer offer) {
```

Comment:
This should be declineOfferShort or somesuch since it's hard coded to use the short decline.

Reply:
We call it declineShort in commons.